licenses/OptiX-SDK

```diff
@@ -0,0 +1,306 @@
+regulations otherwise without first obtaining such export license or approval and NVIDIA’s permission. You will not, without prior
+governmental authorization, export or re-export NVIDIA Licensed Software, technology and related documentation, directly or
+indirectly, (i) to any end-user whom you know or have reason to know will utilize them in the design, development or production of
+nuclear, chemical or biological weapons, or rocket systems, space launch vehicles, and sounding rockets, or unmanned air vehicle
+NVIDIA SDKS, SAMPLES AND TOOLS AGREEMENT (V.07.08.2016)
+
+5
+
+systems; (ii) to any end-user who has been prohibited from participating in U.S. export transactions by any federal agency of the U.S.
+government; or (iii) to any countries that are subject to U.S. export restrictions (currently including, but not necessarily limited to,
+Cuba, Iran, North Korea, Sudan, and Syria and the Region of Crimea).
+8.11
+General. This Agreement constitutes the entire agreement of the parties with respect to the subject matter hereto and
+supersede all prior negotiations, conversations, or discussions between the parties relating to the subject matter hereto, oral or
+written, and all past dealing or industry custom. Any notice delivered by NVIDIA to you under this Agreement will be delivered via
+mail, email or fax. Any additional and/or conflicting terms and conditions on purchase order(s) or any other documents issued by you
+are null, void, and invalid. This Agreement and the rights and obligations hereunder may not be assigned by you, in whole or in part,
+including by merger, consolidation, dissolution, operation of law, or any other manner, without written consent of NVIDIA, and any
+purported assignment in violation of this provision shall be void and of no effect. Each party acknowledges and agrees that the other
+is an independent contractor in the performance of this Agreement, and each is solely responsible for all of its employees, agents,
+contractors, and labor costs and expenses arising in connection therewith. This Agreement will be governed by and construed under
+the laws of the State of Delaware and the United States without regard to the conflicts of law provisions thereof and without regard
+to the United Nations Convention on Contracts for the International Sale of Goods. The parties consent to the personal jurisdiction
+of the federal and state courts located in Santa Clara County, California. If one or more provisions of this Agreement are held to be
+unenforceable under applicable law, such provision shall be excluded from this Agreement and the balance of this Agreement shall
+be interpreted as if such provision were so excluded and shall be enforceable in accordance with its terms. Any amendment or
+waiver under this Agreement must be in writing and signed by representatives of both parties.
+
+NVIDIA SDKS, SAMPLES AND TOOLS AGREEMENT (V.07.08.2016)
+
+6
+
+
```

Comment:
I'm seeing some form feed control characters in there, github is seeing it as a "no eol at end of file", in vim I see "^L" in a few places.

`tr -d '\f' < OptiX-SDK` should do the job without mangling

Reply:
Thanks for the hint. It removed the page breaks, but I needed to do some formatting manually.